yarn-project/prover-node/src/prover-coordination/factory.ts

```diff
@@ -46,27 +61,31 @@ export async function createProverCoordination(
     if (!deps.archiver || !deps.worldStateSynchronizer || !deps.telemetry || !deps.epochCache) {
       throw new Error('Missing dependencies for p2p prover coordination');
     }
-
-    const proofVerifier = config.realProofs ? await BBCircuitVerifier.new(config) : new TestCircuitVerifier();
-    const p2pClient = await createP2PClient(
-      P2PClientType.Prover,
-      config,
-      deps.archiver,
-      proofVerifier,
-      deps.worldStateSynchronizer,
-      deps.epochCache,
-      deps.telemetry,
-    );
-    await p2pClient.start();
-
-    return p2pClient;
   }
```

Comment:
Seems like we're now setting up p2p coordination always? Should we remove the `p2pEnabled` flag then?

Reply:
We are always using the `P2PClient`, as it contains the tx pool. The `p2pEnabled` config still determines whether or not the `P2PClient` actually uses the p2p network.

The tx pool should probably be refactored out of the `P2PClient`, but that's a big job I suspect.